src/frontend/shared-components/src/components/table/Table.test.js

```diff
@@ -2,16 +2,17 @@ import React from "react";
 import testBasicSnapshot from "../../TestHelper";
 
 import Table from "./Table";
-import { getTableTestData } from "../../modules/tableTestData";
+import { getTableElementsTestData } from "../../modules/tableTestData";
 
-const tableData = getTableTestData();
+const tableData = getTableElementsTestData();
+const header = "BCeID Info";
 
 describe("Table Component", () => {
   test("Matches the default snapshot", () => {
-    testBasicSnapshot(<Table table={tableData} />);
+    testBasicSnapshot(<Table heading={header} elements={tableData} />);
```

Comment:
same here:
can we add `styling` to at least one of these? so that we have a snapshot where className is not ""
same for `id`